testing/sdk_cmd.py

```diff
@@ -69,6 +70,7 @@ def cluster_request(
     :param retry: Whether to retry the request automatically if an HTTP error (>=400) is returned.
     :param raise_on_error: Whether to raise a `requests.exceptions.HTTPError` if the response code is >=400.
                            Disabling this effectively implies `retry=False` where HTTP status is concerned.
+    :param log_output: Whether to log messages at all. Can be disabled to reduce noise.
```

Comment:
An alternative is to use a different log level (`DEBUG` for the extra things?)

Reply:
A `log_response` parameter was added on master: https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_cmd.py#L74

I just merged in the latest master and used that instead: https://github.com/mesosphere/dcos-commons/pull/2511/commits/e2e9221c687817a7283d610232b2141ae8320aed